projects/igniteui-angular/src/lib/core/styles/components/charts/_financial-chart-theme.scss

```diff
@@ -0,0 +1,279 @@
+            subtitle-margin: $subtitle-margin,
+            subtitle-alignment: $subtitle-alignment,
+            brushes: $brushes,
+            outlines: $outlines,
+            marker-brushes: $marker-brushes,
+            marker-outlines: $marker-outlines,
+            plot-area-background: $plot-area-background,
+            indicator-brushes: $indicator-brushes,
+            indicator-negative-brushes: $indicator-negative-brushes,
+            indicator-thickness: $indicator-thickness,
+            negative-brushes: $negative-brushes,
+            negative-outlines: $negative-outlines,
+            overlay-brushes: $overlay-brushes,
+            overlay-outlines: $overlay-outlines,
+            overlay-thickness: $overlay-thickness,
+            thickness: $thickness,
+            toolbar-height: $toolbar-height,
+            trend-line-brushes: $trend-line-brushes,
+            trend-line-thickness: $trend-line-thickness,
+            volume-brushes: $volume-brushes,
+            volume-outlines: $volume-outlines,
+            volume-thickness: $volume-thickness,
+            x-axis-label-margin: $x-axis-label-margin,
+            x-axis-label-text-color: $x-axis-label-text-color,
+            x-axis-label-vertical-alignment: $x-axis-label-vertical-alignment,
+            x-axis-major-stroke: $x-axis-major-stroke,
+            x-axis-major-stroke-thickness: $x-axis-major-stroke-thickness,
+            x-axis-minor-stroke: $x-axis-minor-stroke,
+            x-axis-minor-stroke-thickness: $x-axis-minor-stroke-thickness,
+            x-axis-strip: $x-axis-strip,
+            x-axis-stroke: $x-axis-stroke,
+            x-axis-stroke-thickness: $x-axis-stroke-thickness,
+            x-axis-title-alignment: $x-axis-title-alignment,
+            x-axis-title-margin: $x-axis-title-margin,
+            x-axis-title-text-color: $x-axis-title-text-color,
+            y-axis-label-margin: $y-axis-label-margin,
+            y-axis-label-text-color: $y-axis-label-text-color,
+            y-axis-label-vertical-alignment: $y-axis-label-vertical-alignment,
+            y-axis-major-stroke: $y-axis-major-stroke,
+            y-axis-major-stroke-thickness: $y-axis-major-stroke-thickness,
+            y-axis-minor-stroke: $y-axis-minor-stroke,
+            y-axis-minor-stroke-thickness: $y-axis-minor-stroke-thickness,
+            y-axis-strip: $y-axis-strip,
+            y-axis-stroke: $y-axis-stroke,
+            y-axis-stroke-thickness: $y-axis-stroke-thickness,
+            y-axis-title-alignment: $y-axis-title-alignment,
+            y-axis-title-margin: $y-axis-title-margin,
+            y-axis-title-text-color: $y-axis-title-text-color,
+            zoom-slider-x-axis-major-stroke: $zoom-slider-x-axis-major-stroke,
+            zoom-slider-x-axis-major-stroke-thicknes: $zoom-slider-x-axis-major-stroke-thickness
+        )
+    );
+}
+
+@mixin igx-financial-chart-typography($type-scale, $categories: (
+    title: 'h1',
+    subtitle: 'subtitle-1',
```

Comment:
Quite a few text-related category styles missing. Take a look at the category/shape chart.